src/game.h

```diff
@@ -32,7 +32,9 @@
 #include "lib/framework/vector.h"
 #include "gamedef.h"
 #include "levels.h"
-
+#include <3rdparty/json/json.hpp>
```

Comment:
Because of naming conflicts, one of two changes must be made here:
- `json.hpp` needs to either be before WZ includes (but this is hard to accomplish for header files included elsewhere), or there's a undef/re-def dance that can be used (a few of the other files have it)
- `#include <3rdparty/json/json_fwd.hpp>` should be used instead (this is generally the easier path for header files)